src/basic/memory-util.h

```diff
@@ -12,9 +12,12 @@
 #include "memory-util-fundamental.h"
 
 size_t page_size(void) _pure_;
-#define PAGE_ALIGN(l) ALIGN_TO((l), page_size())
-#define PAGE_ALIGN_DOWN(l) ((l) & ~(page_size() - 1))
-#define PAGE_OFFSET(l) ((l) & (page_size() - 1))
+#define PAGE_ALIGN(l)         ALIGN_TO(l, page_size())
+#define PAGE_ALIGN_64(l)      ALIGN_TO_64(l, page_size())
```

Comment:
so looking at this one might get the idea this was aligning to 64bit multiples. Which it of course is not. But if we rename this to PAGE_ALIGN_U64() then I think everything is much clearer, as everyone knows that u64 refers to a unsigned in widely.